prep-css-flexbox/flex-wrap.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+  /* your styles here 👇 */
+  #card-deck {
+    display: flex;
+    flex-wrap: wrap;
+    flex-basis: calc(100%/6);
```

Comment:
Hi Daniel, for each of your pages that need flex-basis, try moving the flex-basis to the rule set for the `.card-container`.

Reply:
Thank You, it worked and now should be correct.